draft-ietf-quic-recovery.md

```diff
@@ -1031,9 +1031,8 @@ OnAckReceived(ack, pn_space):
     OnPacketAcked(acked_packet.packet_number, pn_space)
 
   DetectLostPackets(pn_space)
```

Comment:
I propose we change this to be as follows, keeping with the discussion on the issue.
```
if (DetectLostPackets(pn_space)):
    SendOnePacket()
SendPacketsAsCongestionControllerAllows()
```

Reply:
Q: Why would you always send one packet?  Depending on how ACKs were grouped, this could result in an unnecessary burst of packets that could defeat the congestion controller's intent.  Based on my experience, I don't think this is a purely theoretical problem.

Reply:
Detection of a lost packet at a minimum signals that one packet left the network. This is the logic by which TCP sends a packet immediately on loss detection. It allows a sender to recover from a single packet loss immediately. Do you think we should deviate from what TCP does here?

Reply:
I want to keep when to send(congestion control) and loss detection separate.  Stating that you always send one packet in this pseudocode commingles the two.

Also, we have no normative text that indicates sending a packet after receiving an ACK is the right thing to do here, so putting it in the pseudocode would make the pseudocode out of sync with the text.  If you think we should allow sending a packet because TCP does, then the right thing to do is file an issue and then write a PR that updates the text and the pseudocode.